src/core/mavsdk_impl.h

```diff
@@ -15,6 +15,19 @@ namespace mavsdk {
 
 class MavsdkImpl {
 public:
+    /** @brief Default System ID for GCS configuration type. */
```

Comment:
Docstrings are not required for impl headers. I think they are just destracting. Good variable naming is enough.